std/build.zig

```diff
@@ -904,6 +892,15 @@ pub const Builder = struct {
     }
 };
 
+test "builder.findProgram compiles" {
```

Comment:
Note these kind of tests can be as simple as:

```zig
comptime {
    _ = Builder.findProgram;
}
```
This will reference the function and cause it to be compiled.